packages/node/src/config.ts

```diff
@@ -73,3 +72,46 @@ export const Defaults: ExperimentConfig = {
   fetchRetryBackoffScalar: 1.5,
   fetchRetryTimeoutMillis: 10000,
 };
+
+/**
+ * Configuration for the {@link LocalEvaluationClient}
+ *
+ * @category Configuration
+ */
+export type LocalEvaluationConfig = {
```

Comment:
does the api key also belong to this?

Reply:
API key is injected into factory method/constructor. The general rule is that only fields that have a default value are configurations. Things like api key which does not have a default value are not configuration options.